package/files/nginx-pubcloud/nginx-http.conf

```diff
@@ -0,0 +1,36 @@
+server {
+    listen 80   default;
+    server_name rmt;
+    access_log  /var/log/nginx/rmt_http_access.log;
+    error_log   /var/log/nginx/rmt_http_error.log;
+    root        /usr/share/rmt/public;
+
+    include /etc/nginx/rmt-auth*.d/auth-location*.conf;
+
+    location / {
+        autoindex off;
+    }
+
+    location /repo {
+        return 301 https://$host$request_uri;
+    }
+
+    location = /repo/repoindex.xml {
+        return 301 https://$host$request_uri;
+    }
+
+    location /suma {
+        return 301 https://$host$request_uri;
```

Comment:
How does $host get resolved here? Are we sure we will end up with, for example smt-ec2.susecloud.net? This construct is the only DNS name that should ever be visible to the client.

Reply:
[`$host`](http://nginx.org/en/docs/http/ngx_http_core_module.html#var_host) should basically contain whatever hostname/IP the client has requested.

Reply:
The `/suma` endpoint would potentially be useful for SUMa on-demand. However, at this point there are no considerations to offer SUMa on demand thus for now it could certainly be dropped.